tools/shaderc/shaderc.cpp

```diff
@@ -960,9 +967,18 @@ namespace bgfx
 			{
 				pssl = 1;
 			}
-			else if (0 == bx::strCmp(profile, "spirv") )
+			else if (0 == bx::strCmp(profile, "spirv1-11", 9) )
+			{
+				spirv = 11;
+			}
+			else if (0 == bx::strCmp(profile, "spirv1-12", 9) )
+			{
+				spirv = 12;
+			}
+			else if (0 == bx::strCmp(profile, "spirv1-10", 9)
+			      || 0 == bx::strCmp(profile, "spirv", 5) )
```

Comment:
These strCmp should be without length argument. Those need to match 100%, it's not prefix.

Reply:
Sweet, no dramas. I was just following the leader of the dx shader model profiles, but stopped short of the `&profile[0]`.. 
Would you be okay with me cleaning that bit up as well, while I'm here ?

Reply:
No dramas.